### ISSUE TITLE
Are there known forks for this project? it looks dead

### ISSUE THREAD

**Issue by ghost** [jely2002/youtube-dl-gui#439]
It looks it's not maintained anymore since March of this year.

**Comment by StefanLobbenmeier** [comment]
I did at least one release here: https://github.com/StefanLobbenmeier/youtube-dl-gui/releases

But it would be nice to have this fork more official, right now the same issue is reported once a week because noone can find that fork 😄

**Comment by helpthemfix** [comment]
> I did at least one release here: https://github.com/StefanLobbenmeier/youtube-dl-gui/releases
> 
> But it would be nice to have this fork more official, right now the same issue is reported once a week because noone can find that fork 😄

Thanks, I will check it out. Does have up to date dependencies?

**Comment by StefanLobbenmeier** [comment]
If you know what code changes are needed, I would greatly appreciate a PR for updated dependencies and especially node version. But I do not know node.js that well so I am struggling with it a bit. You can use this as  a starting point: https://github.com/StefanLobbenmeier/youtube-dl-gui/tree/update-dependencies

**Comment by helpthemfix** [comment]
> If you know what code changes are needed, I would greatly appreciate a PR for updated dependencies and especially node version. But I do not know node.js that well so I am struggling with it a bit. Updating dependencies is easy, figuring out how to migrate code is hard. You can use this as a starting point: https://github.com/StefanLobbenmeier/youtube-dl-gui/tree/update-dependencies

Sadly I have no coding experience. but thank you so much for the updated Binaries. it's much faster than the previous one!

**Comment by StefanLobbenmeier** [comment]
Faster than 2.3.1 right? Cause the speed should be the same as 2.4.0

**Comment by StefanLobbenmeier** [comment]
@olegshulyakov we should probably sync up 😄 if you want I can also help you create releases, I saw that you already bumped a lot of dependencies which I struggled with

**Comment by helpthemfix** [comment]
> Faster than 2.3.1 right? Cause the speed should be the same as 2.4.0

Yes faster than  2.3.1 up to more than 80%. Maybe because many dependencies were ancient in 2.3.1.

**Comment by jely2002** [comment]
Thanks all for providing maintained forks. I'll be closing this issue now as v3 has been released.
